RELEASING.md

```diff
@@ -16,18 +36,18 @@
    git push origin X.Y.Z
    ```
 5. Wait for the [publishing workflow](https://github.com/adevinta/spark-android/actions/workflows/publish.yml) to build and publish the release.
-6. Update the `version` in [gradle.properties](gradle.properties) to the next `SNAPSHOT` version.
+6. Update the `version` in [gradle.properties](https://github.com/adevinta/spark-android/blob/main/gradle.properties) to the next `SNAPSHOT` version.
```

Comment:
same

Reply:
Done in https://github.com/adevinta/spark-android/commit/6730295a22b486d089a1d554ba9e1e2add81719b